lib/recipes/nursing-external/strategy.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+# Contemporary answer key strategy
+class Strategy
+  def bake(chapter:, append_to:)
+    # Hacky numbering fix
+    Kitchen::Directions::MoveSolutionsFromNumberedNote.v1(
```

Comment:
So they're numbered in order? Would it help to make an array called classes of the ones you want to use and then use classes.each?